docs/config/README.md

```diff
@@ -2407,20 +2407,21 @@ By default the module will be shown if any of the following conditions are met:
 
 ### Options
 
-| Option              | Default                              | Description                                                               |
-| ------------------- | ------------------------------------ | ------------------------------------------------------------------------- |
-| `format`            | `"via [$symbol$stack]($style) "`     | The format string for the module.                                         |
-| `version_format`    | `"v${raw}"`                          | The version format. Available vars are `raw`, `major`, `minor`, & `patch` |
-| `symbol`            | `" "`                                | A format string shown before the Pulumi stack.                          |
-| `style`             | `"bold 5"`                           | The style for the module.                                                 |
-| `disabled`          | `false`                              | Disables the `pulumi` module.                                             |
+| Option              | Default                                          | Description                                                               |
+| ------------------- | ------------------------------------------------ | ------------------------------------------------------------------------- |
+| `format`            | `"via [$symbol($username@)$stack]($style) "`     | The format string for the module.                                         |
+| `version_format`    | `"v${raw}"`                                      | The version format. Available vars are `raw`, `major`, `minor`, & `patch` |
+| `symbol`            | `" "`                                            | A format string shown before the Pulumi stack.                          |
+| `style`             | `"bold 5"`                                       | The style for the module.                                                 |
+| `disabled`          | `false`                                          | Disables the `pulumi` module.                                             |
 
 ### Variables
 
 | Variable | Example    | Description                          |
 | -------- | ---------- | ------------------------------------ |
 | version  | `v0.12.24` | The version of `pulumi`              |
 | stack    | `dev`      | The current Pulumi stack             |
+| username | `foo`      | The current Pulumi username          |
```

Comment:
Can we use a slightly more descriptive example here?

Reply:
Sure, this would be any (I'd have to check) url safe string though, for example `friel`, or `chipbuster`.

Reply:
Sure, that sounds good. Or we could come up with a dummy username like `admin` or `user`, just something that carries some connotations with it.

Reply:
/shrug Changed to `alice`